### ISSUE TITLE
Can you explain something about Reweighting(RSL loss)?

### ISSUE THREAD

**Issue by hhjung1202** [YisenWang/Iterative_learning#5]
When i read the paper which you wrote, I have a question about Reweighting.

![default](https://user-images.githubusercontent.com/17779678/53636965-37f9ec00-3c65-11e9-9701-90a0d3e50d3c.PNG)

According to your paper, as GAMMA is closed to 1, Input probably have Clean Label. and if else, Input probably have Noisy Label. That's because Gamma is 1 - pcLOF.

When i think about P(Y_j | X_j, Theta) which is detected noisy samples probability, I cannot understand what you intend to. 

![kakaotalk_20190301_220432837](https://user-images.githubusercontent.com/17779678/53639965-20733100-3c6e-11e9-99fc-f8e223eff485.jpg)
if gamma and P are 0.9, then loss size will be low value.
if gamma is 0.9 and P is 0.1, then loss size will be high value.
So far as I think.
but if gamma and P are 0.1, then i think loss would be low value. but it isn't.
also gamma is 0.1 and P is 0.9, then i think loss would be high value. but it isn't.

at this paper, you said that the detected noisy samples may contain some clean sample. so you add RSL loss function. but Y_j is not "Clean Label" but "detected noisy label". and you don't know what is real label. In this situation that function cannot fix the wrong thing i think.

If Y_j is Clean Label, I understand what you intend to. but it would be costly labeling work

**Comment by YisenWang** [comment]
Maybe you have a misunderstanding here. P(y|x) is the output probability of NN not a noisy label probability. The reweighting is doing like this, samples are feeding into the network, for detected noisy label example, we give a small weight before its loss function.

**Comment by hhjung1202** [comment]
Oh thanks for your reply.
At your reply. you said that P(y|x) is the output probability of NN not a noisy label probability.
then i think, you should pretrained NN-model to extract P(y|x). am i right?

**Comment by YisenWang** [comment]
No need for pretrain. When forwarding the NN, you will get one output, one probability.

**Comment by hhjung1202** [comment]
I'm impressive to your papers and thanks for your answer XD.

I have another two question.

First Question.

LOF value is relative value with other input.

![image](https://user-images.githubusercontent.com/17779678/53732033-f7e27580-3ebf-11e9-98e3-314d2b543bfb.png)

When you get the pcLOF value, 
Did you get the LOF value using all the input data?


Second Question.

You said that P(y|x) is the output of NN. then where is that NN??

![default](https://user-images.githubusercontent.com/17779678/53732123-3e37d480-3ec0-11e9-8df2-a96163973e35.PNG)

Is that NN right?
